big_scape/genbank/gbk.py

```diff
@@ -396,14 +396,25 @@ def parse(
         if int(as_version[0]) >= 5:
             gbk.parse_as5up(record, cds_overlap_cutoff)
         if int(as_version[0]) == 4:
-            gbk.parse_as4(record, cds_overlap_cutoff)
+            # if regions are missing, gbk will be assumed to be as4
+            # here we need to see if the user set --gbk-force to true
+
+            if run["force_gbk"]:
+                logging.warning(
```

Comment:
Maybe move this warning to line 472 instead, when it actually is applied. With a mix of valid AS4 and invalid files this would trigger on valid AS4 as well.